ValidationLibrary.Rules/HasNotManyStaleBranchesRule.cs

```diff
@@ -77,6 +76,15 @@ public async Task<ValidationResult> IsValid(IGitHubClient client, Repository git
             return new ValidationResult(RuleName, "Remove branches, that have not been updated in 90 days or more.", staleCount < StaleCountLimit, DoNothing);
         }
 
+        public Dictionary<string, string> GetConfiguration()
+        {
+            return new Dictionary<string, string>
+            {
+                 { "PullRequestTitle", RuleName },
```

Comment:
This rule doesn't create a Pull Request, so `PullRequestTitle` doesn't make sense here